### PR TITLE
sci-biology/ABI_expression_data: new ebuild

### PR DESCRIPTION
**Pull request by setina42**
New ebuild for packaged gene expression data from the Allen Mouse Brain data Portal.
Package-Manager: Portage-2.3.54, Repoman-2.3.12

### CODE REVIEW

**Comment by setina42** [comment]
@TheChymera I have renamed the ebuild as you suggested.

**Comment by TheChymera** [comment]
@setina42 please delete the duplicated files with the old naming scheme, and rebase everything into one commit `rebase -i HEAD~2`.

File: sci-biology/ABI-expression-data/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+		<email>tina.seg@bluewin.ch</email>
+		<name>Tina Segessemann</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>sci@gentoo.org</email>
+		<name>Gentoo Science Project</name>
+	</maintainer>
+	<longdescription lang="en">
+		Gene expression data from the Allen Mouse Brain
+		data portal.
```
Comment (TheChymera) [inline sci-biology/ABI-expression-data/metadata.xml]: again, please add a more detailed description

File: sci-biology/ABI-expression-data/ABI-expression-data-0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit check-reqs
+
+DESCRIPTION="Collection of gene expression data form the Allen Mouse Brain data portal"
```
Comment (TheChymera) [inline sci-biology/ABI-expression-data/ABI-expression-data-0.1.ebuild]: Same issue as before regarding style.

**Comment by TheChymera** [comment]
Merged in https://gitweb.gentoo.org/proj/sci.git/commit/?id=944ce87d136aaaf6b656691fc517c412b88dd50c

**Pull request closed without merging**